src/sdk/ai.ts

```diff
@@ -11,5 +12,16 @@ export default async function ai({ prompt, model }: AiProps) {
     google: Google,
     ollama: Ollama,
   }
-  models[model](prompt)
+
+  if (!models[model]) {
+    throw new Error(`Model ${model} is not supported`)
+  }
+
+  // Ejecuta el modelo seleccionado con el prompt proporcionado
+  await models[model](prompt)
+
+  // Solicita al usuario que ingrese un nuevo prompt
+  const res = await terminalPrompt()
+  bosh(res)
+  term()
```

Comment:
This creates a second shell enviroment, which makes the user need to type 'exit' or 'quit' each time they use the AI feature.

Please change this :(